app/.htaccess

```diff
@@ -1,5 +1,8 @@
 <IfModule mod_rewrite.c>
 	RewriteEngine on
+	# Uncomment if you have a .well-known directory in the app folder, e.g. for the Let's Encrypt challenge
+	# https://tools.ietf.org/html/rfc5785
+	#RewriteRule ^(\.well-known/.*)$ $1 [L]
```

Comment:
I understood (and tested) as if the Apache document root was set either to 
1. CakePHP root
2. app folder
3. webroot (no change needed)